crates/fiber-lib/src/fiber/network.rs

```diff
@@ -1978,20 +1895,29 @@ where
                     .send_command_to_channel(c.channel_id, c.command)
                     .await?
             }
-            NetworkActorCommand::SendPaymentOnionPacket(command) => {
-                if let Err(err) = self
+            NetworkActorCommand::SendPaymentOnionPacket(command, reply) => {
+                match self
                     .handle_send_onion_packet_command(state, command.clone())
                     .await
                 {
-                    self.on_add_tlc_result_event(
-                        myself,
-                        state,
-                        command.payment_hash,
-                        command.attempt_id,
-                        Some((ProcessingChannelError::TlcForwardingError(err.clone()), err)),
-                        command.previous_tlc,
-                    )
-                    .await;
+                    Ok(()) => {
+                        let _ = reply.send(Ok(()));
```

Comment:
the old code seems do not expect get reply from this point.

is there any code really waiting for this reply?
i remember we get a deadlock when some channel actor is waiting for the result from network actor, and network actor is waiting some result from channel actor.